portfolio/pom.xml

```diff
@@ -23,6 +23,27 @@
       <version>1.9.59</version>
     </dependency>
     
+    <dependency>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-testing</artifactId>
+        <version>1.9.59</version>
+        <scope>test</scope>
+    </dependency>
+
+    <dependency>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-api-labs</artifactId>
+        <version>1.9.59</version>
+        <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-api-stubs</artifactId>
+        <version>1.9.59</version>
+        <scope>test</scope>
+    </dependency>
```

Comment:
Keep the indent to two spaces, or indent the rest of the file with four spaces. If you're still using vim, then running `:set cindent shiftwidth=2` for the former (or `:set cindent shiftwidth=4` for the latter) then followed by `gg`, `=`, and `G` should reindent the file for you with consistent indentation.

Reply:
ah ok noted! thank you!